src/main/java/com/slimebot/level/RankCard.java

```diff
@@ -11,13 +16,19 @@
 import java.awt.image.BufferedImage;
 import java.io.IOException;
 import java.net.URL;
+import java.util.Map;
+
+import static com.slimebot.level.profile.Style.ROUND;
```

Comment:
Please don't use a static import here, just use Style.ROUND when the enum constant is used